app/(tabs)/profile.tsx

```
@@ -15,7 +15,9 @@ import * as ImagePicker from 'expo-image-picker';
 // ^ Make sure this file exists, or remove if unneeded
 import { ScreenWrapper } from '../../components/layouts/ScreenWrapper';
 import { Colors } from '../../styles/Theme';
-import { createSupabaseClient } from '../../lib/supabaseClient';
+import { createClient } from '@supabase/supabase-js';
+
+const supabaseUrl = process.env.EXPO_PUBLIC_SUPABASE_URL!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)